modules/candidate_list/test/candidate_listTest.php

```diff
@@ -350,6 +350,7 @@ function testPSCIDLink()
             "document.querySelectorAll('#dynamictable > tbody > tr >".
             " td.dynamictableFrozenColumn > a')[0].click()"
         );
+        sleep(1);
```

Comment:
1. This is not related to the module
2. If you are using this sleep to wait for the new page to load after the click , then `safeClick()` is there especially for that

Reply:
It works on PHP 7.1. Sometimes it doesn't work on PHP 7.0.
This is a react link, using normal safeclick function can't find the react element.